### ISSUE TITLE
how about the pretrained model?

### ISSUE THREAD

**Issue by Marcus208** [ouyanghuiyu/thundernet_mmdetection#1]
Thanks for your great job!

For this project based on mmdetection, could you provide the pre-trained model about the shufflenetv2 1.5x?
